lib/_debugger.js

```diff
@@ -578,7 +578,7 @@ Client.prototype.mirrorObject = function(handle, depth, cb) {
     });
     return;
   } else if (handle.type === 'function') {
-    val = function() {};
+    val = function empty() {};
```

Comment:
@pvsousalima this where the linter fails due to [func-name-matching](http://eslint.org/docs/rules/func-name-matching).